### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by Manishearth**
r? @crlf0710

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -22,6 +22,5 @@ compiler_builtins = { version = "0.1", optional = true }
 
 [features]
 default = []
-no_std = []
```
Comment (crlf0710) [inline Cargo.toml:25]: This is fast! If you want to remove this feature, maybe you want to remove some corresponding comments in lib.rs mod comments too?

File: src/tables.rs [hunk 2]
```diff
@@ -177,23 +177,24 @@ pub mod identifier_status {
         '\u{2dd6}'), ('\u{2dd8}', '\u{2dde}'), ('\u{3005}', '\u{3007}'), ('\u{3041}', '\u{3094}'),
         ('\u{3095}', '\u{3096}'), ('\u{3099}', '\u{309a}'), ('\u{309d}', '\u{309e}'), ('\u{30a0}',
         '\u{30a0}'), ('\u{30a1}', '\u{30fe}'), ('\u{3105}', '\u{312c}'), ('\u{312d}', '\u{312d}'),
-        ('\u{312e}', '\u{312e}'), ('\u{312f}', '\u{312f}'), ('\u{31a0}', '\u{31b7}'), ('\u{31b8}',
```
Comment (crlf0710) [inline src/tables.rs]: this change is strange. According to https://www.unicode.org/Public/security/12.1.0/IdentifierStatus.txt  , \u{312e} is indeed allowed.

**Comment by crlf0710** [comment]
Looks good to me modulo the data change. It seems you're using Unicode 13.0 data somehow?

**Comment by Manishearth** [comment]
Oh, right, my bad, I forgot to delete the old file

**Comment by Manishearth** [comment]
Messed up the merge, did it manually instead.

**Pull request merged**